Apps/APN/ios/MyAppPushNotificationsHandler.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import CioMessagingPushAPN
+import UserNotifications
+
+// This class manages all function calls to CustomerIO SDK.
+@objc
+public class MyAppPushNotificationsHandler : NSObject {
+
+  public override init() {}
+
+  // Skip this function if you don't want to request push permissions on launch
+  @objc(registerPushNotification:)
+  public func registerPushNotification(withNotificationDelegate notificationDelegate: UNUserNotificationCenterDelegate) {
+
+    let center  = UNUserNotificationCenter.current()
+    center.delegate = notificationDelegate
+    center.requestAuthorization(options: [.sound, .alert, .badge]) { (granted, error) in
+      if error == nil{
+        DispatchQueue.main.async {
+          UIApplication.shared.registerForRemoteNotifications()
+        }
+      }
+    }
+  }
```

Comment:
I strongly suggest that our RN sample apps use the SDK's request push permission feature instead of doing this. Since that's our recommended way of requesting push permissions for customers, our sample apps should use the recommended approaches.

Reply:
Oops! My bad. Not using this function anywhere. Instead calling `showPromptForPushNotifications` to show push permission prompt. Thanks for highlighting this.

Reply:
Removed it in [this commit](https://github.com/customerio/customerio-reactnative/pull/136/commits/999139686ef2c79fe835f79fb0a1c3e3cdef8e16).